GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -3416,6 +3421,7 @@ public enum CommitKind
     {
         Normal,
         Fixup,
-        Squash
+        Squash,
+        Amend,
```

Comment:
Must be handled everywhere, e.g. in `OnFormClosed` and `PreserveCommitMessageOnReopen`, too.